anvilPoll2024MainAnalysis.Rmd

```diff
@@ -977,7 +977,38 @@ whereRunPlot
 
 ## Preferences: DMS compliance/data repositories
 
-NEED TO FILL OUT
+```{r}
+dmsPlot <- resultsTidy %>%
+  separate_longer_delim(RepositoriesDMS, delim=", ") %>%
+  separate_longer_delim(RepositoriesDMS, delim=" & ") %>%
+  mutate(RepositoriesDMS = 
+           recode(RepositoriesDMS,
+                  "dbGaP" = "dbGaP*",
```

Comment:
a `regex` with `ignore_case = TRUE` could be more concise here https://stringr.tidyverse.org/reference/modifiers.html

Reply:
Also utilized some codebook mining, a left_join, and a unite to remove the other recode steps